### ISSUE TITLE
node view does not work for parameters

### ISSUE THREAD

**Issue by uvchik** [oemof/oemof#475]
I heard that the `param_results()` function produces the same structure as the `results()` function. But the following code does not work.

```python
results = outputlib.processing.results(model)
param = outputlib.processing.param_results(model)
res_view = outputlib.views.node(results, 'my_bus')
param_view = outputlib.views.node(param, 'my_bus')
```

The last line throws out the following error:
`AttributeError: 'dict' object has no attribute 'empty'`

If I use a different code (the key instead of the label) I do not get an error anymore but an empty dictionary.
```python
>>> outputlib.views.node(params, ('my_bus', None))
{}
```
Do I think the wrong way or is this a bug.

**Comment by henhuy** [comment]
I just looked into code and found underlying problem - 
We have different data types in `results` and `param_results`:
In `results`, `scalars` and `sequences` are `pandas.Series` or `pandas.DataFrame` respectively.
In `param_results` we made it dicts...
oh no

**Comment by uvchik** [comment]
> So I think, we have to change param_results to use pandas.

I agree because it would be nice to have the same structure for both.

In the next step we could add a merge_results function.

`all_results = merge_results(results, param_resutls)`

Or an `parameter`attribute to the results function.

`all_results = resutls(model, parameter=True)`

What do you think?

**Comment by uvchik** [comment]
Solved with PR #494.
